### PR TITLE
Changes for UXP v7.3 Docs

### PR DESCRIPTION
**Pull request by bhatiaPankaj**
Adding the React-starter for  SWC as per the template

### CODE REVIEW

File: swc-uxp-react-starter/manifest.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "swc-uxp-react-starter",
+  "id": "swc-uxp-react-starter",
+  "version": "1.0.0",
+  "main": "index.html",
+  "manifestVersion": 4,
```
Comment (jardicc) [inline swc-uxp-react-starter/manifest.json:6]: I think you should use version: 5
"featureFlags" is not valid property in manifest version 4 ...according to schema

File: swc-uxp-react-starter/README.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+You can use any of the supported components listed in [our docs](https://developer.adobe.com/photoshop/uxp/2022/uxp-api/reference-spectrum/swc) by following these steps:
+
+1. Use `yarn add` to install the components and its supported version.
+
+```javascript
+yarn add @swc-uxp-wrappers/link;
+yarn add @swc-react/link@0.14.1-react.3029;
+```
+
+**Note:** The UXP-SWC components are delivered via wrappers over specific SWC versions. For example, `@swc-uxp-wrappers/menu` is locked and wrapped on  **0.16.9** version of ```@spectrum-web-components/menu```. Thereby for react framework we will need to use ```@swc-react/menu:0.16.9-react.3029``` which is the react-wrapper closest to the SWC component version (0.16.9) in dependency block too. Also, please note that components like `icons`,`icons-workflow`, `icons-ui`, `theme`, `shared` , `base`,`styles` do not use wrappers and therefore must be directly consumed from `@spectrum-web-components` library npm.
```
Comment (padmkris123) [inline swc-uxp-react-starter/README.md:49]: Typo. react -> React.

File: swc-uxp-react-starter/README.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+import { Link } from "@swc-react/link";
+```
+
+3. Add the component in the same file.
+
+```html
+   This is a <Link href="#">example link</Link>.
+```
+
+4. We need to put alias configuration in the webpack config file. This project contains the configuration already. Notice the `@swc-uxp-wrappers/utils` entry in the package.json. This package delivers the `alias.js` file for all the supported components which is then imported in the `webpack.config.js` file as this.
```
Comment (padmkris123) [inline swc-uxp-react-starter/README.md:66]: put -> add

File: swc-uxp-react-starter/README.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+```
+
+3. Add the component in the same file.
+
+```html
+   This is a <Link href="#">example link</Link>.
+```
+
+4. Add the alias configuration in the Webpack config file.
+This project contains the configuration already. Notice the `@swc-uxp-wrappers/utils` entry in the package.json. This package delivers the `alias.js` file for all the supported components which is then imported in the `webpack.config.js` file as this.
```
Comment (padmkris123) [inline swc-uxp-react-starter/README.md:67]: Do we need to include alias.js?

File: swc-uxp-react-starter/README.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+        <br />
+        This is an <Link href="#">example link</Link>.
+    </div>
+  );
+}
+
+```
+
+
+## Recommended `@swc-react` library  versions
```
Comment (padmkris123) [inline swc-uxp-react-starter/README.md:165]: Can we publish this list in our public docs and provide the link here? Will avoid duplicates and maintenance.

File: swc-uxp-react-starter/package.json [hunk 6]
```diff
@@ -0,0 +1,105 @@
+{
+  "name": "react-starter",
```
Comment (padmkris123) [inline swc-uxp-react-starter/package.json:2]: match it with the name of the project.

File: swc-uxp-react-starter/src/index.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import React from "react";
+import ReactDOM from "react-dom/client";
+// import "./index.css";
```
Comment (padmkris123) [inline swc-uxp-react-starter/src/index.js:15]: Remove

File: swc-uxp-react-starter/README.md (hunk 4 above)
Comment (bhatiaPankaj) [reply swc-uxp-react-starter/README.md:67]: It is included in @swc-uxp-wrappers/utils package

File: swc-uxp-react-starter/README.md (hunk 2 above)
Comment (bhatiaPankaj) [reply swc-uxp-react-starter/README.md]: Done.

File: swc-uxp-react-starter/README.md (hunk 3 above)
Comment (bhatiaPankaj) [reply swc-uxp-react-starter/README.md]: Done

File: swc-uxp-react-starter/README.md (hunk 4 above)
Comment (bhatiaPankaj) [reply swc-uxp-react-starter/README.md]: Done

File: swc-uxp-react-starter/README.md (hunk 5 above)
Comment (bhatiaPankaj) [reply swc-uxp-react-starter/README.md]: Added the link here

File: swc-uxp-react-starter/package.json (hunk 6 above)
Comment (bhatiaPankaj) [reply swc-uxp-react-starter/package.json]: Done

File: swc-uxp-react-starter/src/index.js (hunk 7 above)
Comment (bhatiaPankaj) [reply swc-uxp-react-starter/src/index.js]: Done

**Pull request merged**